FetchMigration/README.md

```diff
@@ -1,16 +1,30 @@
 # Index Configuration Tool
 
-Python package that automates the creation of indices on a target cluster based on the contents of a source cluster. Index settings and index mappings are correctly copied over.
+Python package that automates the creation of indices on a target cluster based on the contents of a source cluster. 
+Index settings and index mappings are correctly copied over, but no data is transferred. 
+This tool seeks to eliminate the need to [specify index templates](https://github.com/awslabs/logstash-output-amazon_es#optional-parameters) when migrating data from one cluster to another.
+The tool currently supports ElasticSearch or OpenSearch as source and target.
 
-This tool seeks to automate [the steps outlined here](https://github.com/kartg/opensearch-migrations/tree/datastash/datastash#2-configure-index-templates-on-the-target-cluster) and eliminate the need for defining index templates on the target cluster when migrating data from one cluster to another. The tool currently supports ElasticSearch or OpenSearch as source and target.
+## Parameters
 
-If an identical index or an index with the same name but differing settings/mappings is already present on the target cluster, that index is skipped and left as-is on the target cluster. The tool completes by printing a report of all processed indices under 3 buckets:
+The first required input to the tool is a path to a [Data Prepper](https://github.com/opensearch-project/data-prepper) pipeline YAML file, which is parsed to obtain the source and target cluster endpoints.
+The second required input is an output path to which a modified version of the pipeline YAML file is written.
+This version of the pipeline adds an index inclusion configuration to the sink, specifying only those indices that were created by the index configuration tool.
+The tool also supports several optional flags:
+
+| Flag | Purpose |
+| ------------- | ------------- |
+| `-h, --help`    | Prints help text and exits |
+| `--report, -r`  | Prints a report of indices indicating which ones will be created, along with indices that are identical or have conflicting settings/mappings.  |
+| `--dryrun`      | Skips the actual creation of indices on the target cluster |
```

Comment:
I'm wary of using the word "simulate" since the dryrun code path doesn't actually call the create_indices method. That's why i used "skip" instead. 

As for the report, I wanted to enable the tool to output a YAML file in a dryrun without printing a report 😄 

Let me know if you think any of this is a bad idea?